web/src/components/countrytableoverlayifnodata.js

```diff
@@ -26,6 +27,20 @@ const CountryTableOverlayIfNoData = ({ zoneTimeIndex }) => {
       <div className="no-data-overlay-background" />
       <div className="no-data-overlay-message">
         {__(isRealtimeData ? 'country-panel.noLiveData' : 'country-panel.noDataAtTimestamp')}
+        {!isRealtimeData ? null : (
+          <React.Fragment>
+            {'. '}
+            {'Help us identify the problem by taking a look at the '}
+            <a
+              href={`https://kibana.electricitymap.org/app/kibana#/discover/10af54f0-0c4a-11e9-85c1-1d63df8c862c?_g=(refreshInterval:('$$hashKey':'object:232',display:'5%20minutes',pause:!f,section:2,value:300000),time:(from:now-24h,mode:quick,to:now))&_a=(columns:!(message,extra.key,level),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!t,index:'96f67170-0c49-11e9-85c1-1d63df8c862c',key:level,negate:!f,params:(query:ERROR,type:phrase),type:phrase,value:ERROR),query:(match:(level:(query:ERROR,type:phrase)))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'96f67170-0c49-11e9-85c1-1d63df8c862c',key:extra.key,negate:!f,params:(query:${zoneId},type:phrase),type:phrase,value:${zoneId}),query:(match:(extra.key:(query:${zoneId},type:phrase))))),index:'96f67170-0c49-11e9-85c1-1d63df8c862c',interval:auto,query:(language:lucene,query:''),sort:!('@timestamp',desc))`}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              runtime logs
+            </a>
+            , or contact our data provider.
```

Comment:
Not necessarily a part of this story, but I think it would be good to make _data provider_ a link here, e.g. linking to the data source website?

![image](https://user-images.githubusercontent.com/1216874/81570941-8b0b1b00-93a1-11ea-94b9-b49a5753c64f.png)